### PR TITLE
Oleksandr/chat with pdf evaluate

### PR DESCRIPTION
**Pull request by AlexTeslenko**

### CODE REVIEW

**Review by seekerofsai — APPROVED** [review]

File: flows/chat_with_pdf/evaluate/evaluate.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
+"""Evaluation flow for yaml_basic_flow."""
+import argparse
+from pprint import pprint
+from promptflow.evals.evaluate import evaluate
+from mlops.common.config_utils import MLOpsConfig
+from promptflow.core import AzureOpenAIModelConfiguration
+from promptflow.evals.evaluators import RelevanceEvaluator, CoherenceEvaluator
```
Comment (anastasiyagromova) [inline flows/chat_with_pdf/evaluate/evaluate.py:7]: Just a thought do we want to use AI Studio Evaluation metrices (out of the box) as well.

**Review by anastasiyagromova — APPROVED** [review]
Approved.

File: config/config.yaml [hunk 2]
```diff
@@ -10,6 +10,7 @@ aoai_config:
   aoai_api_type: azure
   aoai_api_version: 2023-07-01-preview
   aoai_api_key: ${AOAI_API_KEY}
+  aoai_deployment_name: aoai
```
Comment (sbaidachni) [inline config/config.yaml]: we don't need it here. deployment name is something that can be different for every model/pipeline and we can have several deployments per flow (if we use several models). Let's move it to the flows section (under your flow pr and dev)

File: config/config.yaml [hunk 3]
```diff
@@ -82,3 +83,23 @@ flow_configs:
     eval_data_path: data/plan_and_execute_eval_data.jsonl
     column_mapping:
       question: ${data.question}
+
+  yaml_chat_with_pdf_pr:
+    standard_flow_path: flows/chat_with_pdf/standard
+    EMBEDDING_MODEL_DEPLOYMENT_NAME: text-embedding-ada-002
+    CHAT_MODEL_DEPLOYMENT_NAME: aoai
+    deployment_name: aoai
```
Comment (sbaidachni) [inline config/config.yaml:91]: in fact, you have it here. So, it's not needed there at all.

File: config/config.yaml [hunk 4]
```diff
@@ -82,3 +83,23 @@ flow_configs:
...
+    connection_name: aoai
+    data_path: flows/chat_with_pdf/data/bert-paper-qna.jsonl
```
Comment (sbaidachni) [inline config/config.yaml]: let's move all data files into data folder in the root

File: config/config.yaml [hunk 5]
```diff
@@ -82,3 +83,23 @@ flow_configs:
...
+    eval_data_path: flows/chat_with_pdf/data/bert-paper-qna-3-line.jsonl
+    column_mapping:
+      text: ${data.text}
+      entity_type: ${data.entity_type}
+      pdf_url: ${data.pdf_url}
+      chat_history: ${data.chat_history}
+      question: ${data.question}
+    eval_column_mapping:
```
Comment (sbaidachni) [inline config/config.yaml]: not sure how we use this section (eval_column_mapping)

File: flows/chat_with_pdf/evaluate/evaluate.py (hunk 1 above)
Comment (AlexTeslenko) [reply flows/chat_with_pdf/evaluate/evaluate.py:7]: I use here buit-in evaluators from promptflow SDK. I think  AI Studio  evaluators can be used only form AI Studio UI. Or it has SDK too?

**Review by sbaidachni — APPROVED** [review]

**Pull request merged**